fido-key-manager/Cargo.toml

```diff
@@ -28,5 +28,5 @@ tracing.workspace = true
 tracing-subscriber.workspace = true
 webauthn-rs-core.workspace = true
 
-[target.'cfg(windows)'.build-dependencies]
-winres = "0.1"
+[build-dependencies]
```

Comment:
Since this is windows-specific, shouldn't it still be limited thus?

Reply:
cargo.toml doesn'tu have a concept of OS though for deps.

Reply:
oh that's right, cross-compile. sorted! 😄